js/commonJs.js

```diff
@@ -0,0 +1,462 @@
+document.addEventListener("DOMContentLoaded", function (e) {
```

Comment:
Yes, good idea. But once we are on this file - I was thinking about moving "createAutocompleteSource" and "addFacetWidget" method to instantsearch, respectively autocomplete.js file. Because logically it belogs there. WDYT?